src/main/java/com/icthh/xm/ms/entity/domain/XmEntity.java

```diff
@@ -100,7 +97,7 @@
      */
     @ApiModelProperty(value = "Additional lateral identification for this entity could be defined by template Example: PO-1 for Product Offering, e-Mail or MSISDN for account, external Id for Order etc). Key is not full unique entity identification. Few entities could have one key, but only one entity could be addressable at one time, as other should be ended by endDate property.", required = true)
     @Column(name = "jhi_key", nullable = false)
-    @Field(index = FieldIndex.not_analyzed, type = FieldType.String)
+    @Field(type = FieldType.Keyword)
```

Comment:
Can it break search?
Maybe analyzer need to be specified.
It's field should  be not analizer, or search will be broken.

will we found by entity by query 'key: d-c-b-a' If key it's UUID with value a-b-c-d?

Reply:
@sergeysenja1992  do you have a test for that reason?